action.yml

```diff
@@ -116,9 +116,21 @@ runs:
         export GITHUB_BASE_REF
         export GITHUB_HEAD_REF
         echo "::endgroup::"
+        set -euo pipefail
         bash ${{ github.action_path }}/src/scripts/fetch_through_merge_base.sh        
+    # Note: `outcome` does not consider `continue-on-error`
+    # Note: fallback-fetch-all is a string, not a boolean, since type is not supported
+    # for inputs on composite actions (only workflows).
+    - id: check_for_failure
+      if: ${{ steps.fetch_through_merge_base.outcome == 'failure' && inputs.fallback-fetch-all == 'false' }}
```

Comment:
Need to use `outcome`, not `conclusion so `continue-on-error: true` doesn't always return `true`